tests/acceptance/addon-works-with-application-hbs-test.js

```diff
@@ -0,0 +1,20 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | addon works with application.hbs', {
+  //beforeEach() {
+  //  $.post('/welcome-add-application-hbs', function(data) {
+  //    console.log(data.status);
+  //  });
```

Comment:
@locks Can I get your input here? Any suggestions on how to test this? Part of me wants to just skip tests for now, but it's pretty high visibility here, don't want things breaking. Any nifty tips/tricks that I'm missing?